CHANGELOG.md

```diff
@@ -2,11 +2,29 @@
 All notable changes to [Solang](https://github.com/hyperledger/solang/)
 will be documented here.
 
-## Unreleased
+## v0.3.2 Brasília
+
+The language server is much improved, and many fixes all over.
+
+### Added
+- Go to definition is now implemented in the language server. [chioni16](https://github.com/chioni16)
+- The parser has been updated to be compatible with Ethereum Solidity v0.8.21. [seanyoung](https://github.com/seanyoung)
 
 ### Fixed
 - **breaking** Resolving import paths now matches solc more closely, and only resolves relative
   paths when specified as `./foo` or `../foo`. [seanyoung](https://github.com/seanyoung)
+- **Solana** The `lamports` and `data` fields of `tx.accounts` can be modified again. [LucasSte](https://github.com/LucasSte)
+- It is not longer necessary to save a Solidity file, in order for the language server to pick
+  up changes to the file. [chioni16](https://github.com/chioni16)
+- The negate operator `-` now checks for overflow at runtime, and other math overflow fixes.
```

Comment:
```suggestion
- The negate operator `-` now checks for overflow at runtime, and other math overflow fixes.
- Fixed a bug in accessing virtual function selectors. [xermicus](https://github.com/xermicus)
- Fixed a bug where the strength reduce optimization pass removed overflow checks on optimized multiplications. [xermicus](https://github.com/xermicus)
- Fixed a bug where external function variables were not marked as `read` when they were called. [xermicus](https://github.com/xermicus)
```

Reply:
- "Fixed a bug in accessing virtual function selectors" is not very descriptive. How does this affect solang users?
- "Fixed a bug where external function variables were not marked as `read` when they were called." this not very clear to our users. What does being marked as "read" actually mean? In what way does this affect them?

Reply:
I explained them a little more detailed